dapla-manual/statistikkere/git-arbeidsflyt.qmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "Anbefalt arbeidsflyt med Git"
+date-modified: 08/28/2025
+---
+
+
+Å lære seg å bruke git handler om å implementere nye rutiner i arbeidshverdagen. Vi kaller dette for *anbefalt arbeidsflyt*. Arbeidsflyten er noe man gjennomfører hyppig iløpet av en arbeidsdag om man jobber med kode. Vi beskriver derfor anbefalt git arbeidsflyt fra man begynner på en ny oppgave til den er avsluttet og merget til main.
+
+Denne siden er en inspirert av Beste Praksis sin confluence-artikkel [*Hvordan er Git anbefalt arbeidsflyt?*](https://statistics-norway.atlassian.net/wiki/spaces/BEST/pages/3206283277/Hvordan+er+anbefalt+git+arbeidsflyt).
+
+
+
+::: {.callout-tip}
+# Usikker på hva Git og GitHub er?
+Les vår artikkel om [Git og GitHub](git-og-github.qmd). Der kan du lære hva verktøyene er og hvorfor vi bruker de i SSB.
+:::
+
+### Arbeidsflyt
+
+#### 1. Oppdater ditt lokale repo
```

Comment:
Du har hoppet over [punkt 1.b.i ](https://statistics-norway.atlassian.net/wiki/spaces/BEST/pages/3206283277/Hvordan+er+anbefalt+git+arbeidsflyt#%5CuD83D%5CuDCD8-Instruksjoner) i den opprinnelige beskrivelsen. Jeg vet ikke om målet er å forenkle, men det punktet ble lagt til fordi vi flere ganger måtte hjelpe personer som fikk problemer fordi de ikke hadde *clean* git status. De hadde typisk endret ting på main, kanskje uten å være klar over det eller at de hadde glemt det. Det er et mindre problem på Daplalab hvor man ofte starter en ny tjeneste og kloner ut repoet på nytt uansett, men husk på at beskrivelsen også skal gjelde for de som jobber i prod-sonen. Og der er det vanlig at man har klonet ut repoet en gang, og jobber videre på det i "evigheter" med ny oppdateringer og brancher.